### PR TITLE
DTGB-748: Added partner block component.

### PR DESCRIPTION
**Pull request by meirege**
## Description
Added new partner block and single partner block variant.

## Screenshots (if appropriate):
Screenshot 2019-09-24 at 11 18 38
Screenshot 2019-09-24 at 11 18 43

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/31-molecules/partner-block/_partner-block.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.partner-block {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
```
Comment (delrueba) [inline components/31-molecules/partner-block/_partner-block.scss]: obsolete, it's already a column direction
```suggestion
```

File: components/31-molecules/partner-block/partner-block.twig [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<div class="partner-block box-no-icon {{ modifier }}">
```
Comment (delrueba) [inline components/31-molecules/partner-block/partner-block.twig]: Is there no semantic tag for this?
Section maybe?

**Review by delrueba — CHANGES_REQUESTED** [review]

File: components/31-molecules/partner-block/partner-block.twig [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      'text': description
+    } %}
+  {% endif %}
+
+
+  {% if single_partner %}
+    <div class="single-partner">
+      {% include '@image' with {
+        src: '/styleguide/img/stock/ocmw-logo.svg' | path,
+        alt_text: 'Partner logo',
```
Comment (delrueba) [inline components/31-molecules/partner-block/partner-block.twig]: You're giving a bit of a bad example here.
Better to just use the company name (wouldn't include 'logo', it's implied).

**Review by delrueba — CHANGES_REQUESTED** [review]

File: components/31-molecules/partner-block/partner-block.twig (hunk 2 above)
Comment (meirege) [reply components/31-molecules/partner-block/partner-block.twig]: Section within a div? Isn't that a bit unnecessary here?

File: components/31-molecules/partner-block/partner-block.twig (hunk 3 above)
Comment (meirege) [reply components/31-molecules/partner-block/partner-block.twig]: Oh okay sure. I thought I'd keep it generic.

File: components/31-molecules/partner-block/partner-block.twig (hunk 2 above)
Comment (delrueba) [reply components/31-molecules/partner-block/partner-block.twig]: DIV doesn't really count, you can have as many wrappers as you like.

I think it's best to make clear that it's a distinct part of whatever semantic element it's used in (I hope footer).

@hstandaert ?

Comment (hstandaert) [reply components/31-molecules/partner-block/partner-block.twig]: Agreed, `section` would more suitable.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this style guide are documented here.
 
 NOTE: Refer to upcoming changes in our README.md under "Roadmap"
 
+## [Not released]
```
Comment (hstandaert) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
```

(should be `Unreleased` to make the link work at the bottom of this file).

File: components/31-molecules/partner-block/partner-block.twig [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    {% include '@paragraph' with {
+      'text': description
+    } %}
+  {% endif %}
+
+
+  {% if single_partner %}
+    <div class="single-partner">
+      {% include '@image' with {
+        src: '/styleguide/img/stock/ocmw-logo.svg' | path,
```
Comment (hstandaert) [inline components/31-molecules/partner-block/partner-block.twig:19]: Would be nice if you made these configurable and looped over them in the template. That'd make the template way more readable in my opinion.

(but that's a nice-to-have)

File: components/31-molecules/partner-block/partner-block.config.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  status: 'alpha',
+  context: {
+    title: 'Partners',
+    description: 'Hier komt een begeleidende tekst bij het logo.'
```
Comment (hstandaert) [inline components/31-molecules/partner-block/partner-block.config.js]: Add some information about the fact that the logo's in this blok are not fixed ?

**Review by hstandaert — CHANGES_REQUESTED** [review]

**Review by delrueba — COMMENTED** [review]
@mattiasberlamont Can we restrict usage to inside the footer tag? Or do you see other possible use cases?

**Comment by hstandaert** [comment]
> @mattiasberlamont Can we restrict usage to inside the footer tag? Or do you see other possible use cases?

This block will not be used in the footer... It's a standalone block on a detail page.

**Review by delrueba — APPROVED** [review]

File: components/31-molecules/partner-block/partner-block.twig (hunk 2 above)
Comment (meirege) [reply components/31-molecules/partner-block/partner-block.twig]: Done.

File: components/31-molecules/partner-block/partner-block.twig (hunk 5 above)
Comment (meirege) [reply components/31-molecules/partner-block/partner-block.twig:19]: Done.

File: components/31-molecules/partner-block/partner-block.config.js (hunk 6 above)
Comment (meirege) [reply components/31-molecules/partner-block/partner-block.config.js]: Done.

**Review by hstandaert — APPROVED** [review]

**Pull request merged**